package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "jigmi",
+  "version": "0.0.0",
+  "description": "Jitsi Generic Monitoring Infrastructure",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/jitsi/jigmi"
+  },
+  "keywords": [],
+  "author": "bbaldino",
+  "readmeFilename": "README.md",
+  "dependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "body-parser": "^1.18.2",
+    "express": "^4.16.2",
+    "highcharts": "^6.0.2",
+    "prop-types": "^15.6.0",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "react-highcharts": "^13.0.0",
+    "react-redux": "^5.0.6",
+    "redux": "^3.7.2",
+    "sqlite3": "^3.1.13",
+    "webpack": "^3.8.1"
```

Comment:
I highly recommend setting packages to a specific version just in case there's a breaking change. It's happened to me. Also, are you sure these are all dependencies and some aren't in devDependencies? Would all these be needed when deploying a server? In all honestly...I don't think it matters that much considering how the repo will be used.

Reply:
ok, this is what `npm install XXX --save` does by default i think.  i can remove the leading carets.  and yeah, some can probably be dev